README.md

```diff
@@ -238,38 +240,27 @@ a single failure.
 
 To learn more about segments, see the [Corfu wiki](https://github.com/CorfuDB/CorfuDB/wiki).
 
-To scale Corfu, we add additional ``stripes''. To add an additional stripe, first 
-start a new ```corfu_server``` on port 9002:
+To add more nodes, begin by 
+starting a new ```corfu_server``` on port 9002:
 ```
 ./CorfuDB/bin/corfu_server -m 9002
 ```
-Redeploy this cluster with the following additions to layout.json:
-The layoutServers line should read:
-```json
-  "layoutServers": [
-    "localhost:9000", "localhost:9001", "localhost:9002"
-  ],
+
+The server can be added to the existing cluster:
+```
+./CorfuDB/bin/corfu_add_node -c localhost:9000,localhost:9001 -n localhost:9002
 ```
-This time we add localhost:9002 as a new stripe.
 
-```json
-      "stripes": [
-        {
-          "logServers": [
-            "localhost:9000",
-            "localhost:9001"
-          ]
-        },
-        {
-          "logServers": [
-            "localhost:9002"
-          ]
-        }
-      ]
+If you now check the layout of the cluster:
+```
+./CorfuDB/bin/corfu_layouts query -c localhost:9000,localhost:9001,localhost:9002
 ```
+You will see that the cluster contains the new node.
 
-This adds the logunit at localhost:9002 as an additional stripe in the system. That is, writes to even addresses will now go to
-localhost:9000 and localhost:9001, while writes to odd addresses will go to localhost:9002.
+To remove the added node from the cluster:
+```
+./CorfuDB/bin/corfu_remove_node -c localhost:9000,localhost:9001,localhost:9002 -n localhost:9002
+```
 
 ## Now I want to write a program that uses Corfu!
 
```

Comment:
I known it's not from your PR. I think there are problems here. 

- The exposed version string for every module in `CorfuDB` repo is `0.3.1-SNAPSHOT`. Currently we are just using SNAPSHOT with timestamp to update our artifacts, which is not visible outside.
- The "You will also want to add the Corfu Maven repository" part might not be valid now. As GitHub Packages have a known [issue](https://github.community/t/download-from-github-package-registry-without-authentication/14407) , authentication are required for even public artifacts. So might be good to give the links for authenticating like [this for Maven](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages) and [this for Gradle](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry).
- For importing dependencies part, I think only adding `runtime` module is not enough, normally we need everything [here](https://github.com/orgs/CorfuDB/packages?repo_name=CorfuDB). How about give the link here, and use `runtime` as an example? The Packages page provides code snippets for importing them actually (after authentication).